docker/docker-compose.yaml

```diff
@@ -15,53 +15,35 @@
 # limitations under the License.
 
 version: '3.8'
+
 services:
-  streampark-console:
+  streampark:
     image: apache/streampark:2.2.0
-    command: ${RUN_COMMAND}
     ports:
-      - 10000:10000
-      - 10030:10030
-    env_file: .env
+      - "10000:10000"
+      - "10030:10030"
+    environment:
+      - TZ=Asia/Shanghai
+      - SPRING_PROFILES_ACTIVE=h2 # h2, mysql, pgsql
```

Comment:
The latest config file is config.yaml , and the setting for the system data source config key: [datasource.dialect](https://github.com/apache/incubator-streampark/blob/d2104a251a5bbcff226f6106c926b5654f52e74d/streampark-console/streampark-console-service/src/main/assembly/conf/config.yaml#L37)... Please correct me if I am wrong

Reply:
Thanks. Fixed.